### ISSUE TITLE
add href attribute for A element in top-app-bar demo

### ISSUE THREAD

**Issue by englishextra** [material-components/material-components-web#2378]
in your demo here https://material-components-web.appspot.com/top-app-bar.html
```html
 <a id="right-item" class="material-icons mdc-top-app-bar__action-item" aria-label="Download" alt="Download">file_download</a>
```
to somewhat of:
```html
 <a href="javascript:void(0);" ...</a>
```
or:
```html
 <a href="#" ...</a>
```

**Comment by patrickrodee** [comment]
Thanks for submitting an issue. We're currently working on a fix for this.

**Comment by kfranqueiro** [comment]
Yup, it's included in #2349 which is nearing completion.

**Comment by englishextra** [comment]
@kfranqueiro @patrickrodee very nice
